packages/arb-token-bridge-ui/src/util/orbitChainsList.json

```diff
@@ -0,0 +1,783 @@
+{
+  "mainnet": {
```

Comment:
It's missing [Mind Network](https://github.com/OffchainLabs/arbitrum-token-bridge/blob/master/packages/arb-token-bridge-ui/src/util/orbitChainsList.ts#L435) that's been added recently